core/src/test/scala/integration/kafka/admin/ReassignPartitionsIntegrationTest.scala

```diff
@@ -151,7 +148,7 @@ class ReassignPartitionsIntegrationTest extends ZooKeeperTestHarness {
       """{"topic":"bar","partition":0,"replicas":[3,2,0],"log_dirs":["any","any","any"]}""" +
       """]}"""
     // Execute the assignment
-    runExecuteAssignment(zkClient, assignment, -1L)
+    runExecuteAssignment(cluster.adminClient, false, assignment, -1L, -1L)
```

Comment:
Looks like the test name doesn't match what we're testing now.

Reply:
Nice catch! The test should also be removed since there's already a test `testReassignment` to test adminClient reassignment. Thank you.